services/cloud/cluster/wssd.go

```diff
@@ -126,7 +126,7 @@ func (c *client) Unload(ctx context.Context, location, name string) error {
 	return err
 }
 
-///////////////////////////
+// /////////////////////////
```

Comment:
Is this change expected ?

Reply:
I think vs code auto format must have done this